examples/lambda/infrastructure/lib/stack.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from 'path';
+import * as cdk from 'aws-cdk-lib';
+import {Construct} from 'constructs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+export class MomentoLambdaStack extends cdk.Stack {
+    constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+        super(scope, id, props);
+
+        // Create Lambda function from Docker Image
+        const dockerLambda = new lambda.DockerImageFunction(this, 'MomentoDockerLambda', {
+            functionName: 'MomentoDockerLambda',
+            code: lambda.DockerImageCode.fromImageAsset(path.join(__dirname, '../../docker')), // Point to the root since Dockerfile should be there
+            environment: {
+                MOMENTO_AUTH_TOKEN: process.env.MOMENTO_AUTH_TOKEN || ''
```

Comment:
should you throw if this env var is not set? The lambda is not going to work so you might save someone some heartache by failing to deploy it?

Reply:
That's a great suggestion. Will do!